x/erc20/keeper/msg_server.go

```diff
@@ -250,7 +250,9 @@ func (k Keeper) convertERC20NativeCoin(
 ) (*types.MsgConvertERC20Response, error) {
 	// NOTE: coin fields already validated
 	coins := sdk.Coins{sdk.Coin{Denom: pair.Denom, Amount: msg.Amount}}
-	erc20 := contracts.ERC20BurnableAndMintableContract.ABI
+	// TODO use native contract abi
```

Comment:
is there an actionable item here? If so, feel free to create an issue 👍

Reply:
@fedekunze That Todo will be part of https://linear.app/tharsis/issue/ENG-396/if-ethermint-06-direct-balance-manipulation-add-test-contract